profiles/management/commands/delete_realistic_search_data.py

```diff
@@ -14,5 +19,12 @@ class Command(BaseCommand):
     help = "Deletes a set of realistic users and programs/courses that were generated to help us test search"
 
     def handle(self, *args, **options):
+        fake_program_ids = [
+            id_dict['id']
+            for id_dict in Program.objects.filter(description__startswith=FAKE_PROGRAM_DESC_PREFIX).values('id')
```

Comment:
You can also do `fake_program_ids = Program.objects.filter(description__startswith=FAKE_PROGRAM_DESC_PREFIX).values_list("id", flat=True)`